<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pe100</class>
 <widget class="QMainWindow" name="pe100">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pe100</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="password_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>User name</string>
    </property>
    <property name="accessibleName">
     <string>User name</string>
    </property>
    <property name="accessibleDescription">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="login_info">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Login In&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="login_user">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="time">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>user_name</tabstop>
  <tabstop>password_1</tabstop>
  <tabstop>password_2</tabstop>
  <tabstop>login</tabstop>
  <tabstop>exit</tabstop>
  <tabstop>login_user</tabstop>
  <tabstop>time</tabstop>
  <tabstop>login_info</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
